.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.1.2
+ruby-2.2.0
```

Comment:
Good point. Will update that shortly. Not gonna update the branch name cause I'd have to open a new PR.